### PR TITLE
Python gRPC/GAPIC Speech

### PR DESCRIPTION
**Pull request by geigerj**
Includes a test script

### CODE REVIEW

File: generated/python/gapic-google-cloud-speech-v1beta1/google/cloud/gapic/speech/v1beta1/speech_api.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+    # Service calls
+    def sync_recognize(self, config, audio, options=None):
+        """
+        Perform synchronous speech-recognition: receive results after all audio
+        has been sent and processed.
+
+        Example:
+          >>> from google.cloud.gapic.speech.v1beta1 import speech_api
+          >>> from google.cloud.speech.v1beta1 import cloud_speech_pb2
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-speech-v1beta1/google/cloud/gapic/speech/v1beta1/speech_api.py]: Need to update path. Use the pattern `s/google\.cloud\(\.[g].*_pb2\)/google.cloud.grpc\1/`.

File: generated/python/gapic-google-cloud-speech-v1beta1/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+googleapis-common-protos>=1.3.4, <2.0.0
+google-gax>=0.14.1, <0.15.0
+# TODO (https://github.com/googleapis/packman/issues/119): need upper bound logic
+grpc-google-cloud-speech-v1beta1>=0.10.0
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-speech-v1beta1/requirements.txt]: Add upper bound here and remove TODO comment

File: generated/python/gapic-google-cloud-speech-v1beta1/setup.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+"""
+
+from setuptools import setup, find_packages
+import sys
+
+install_requires = [
+    'googleapis-common-protos>=1.3.4, <2.0.0',
+    'google-gax>=0.14.1, <0.15.0',
+    # TODO (https://github.com/googleapis/packman/issues/119): need upper bound logic
+    'grpc-google-cloud-speech-v1beta1>=0.10.0',
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-speech-v1beta1/setup.py]: Add upper bound here and remove TODO comment

File: generated/python/gapic-google-cloud-speech-v1beta1/google/cloud/gapic/speech/v1beta1/speech_api.py (hunk 1 above)
Comment (geigerj) [inline generated/python/gapic-google-cloud-speech-v1beta1/google/cloud/gapic/speech/v1beta1/speech_api.py]: Done

File: generated/python/gapic-google-cloud-speech-v1beta1/requirements.txt (hunk 2 above)
Comment (geigerj) [inline generated/python/gapic-google-cloud-speech-v1beta1/requirements.txt]: Done

File: generated/python/gapic-google-cloud-speech-v1beta1/setup.py (hunk 3 above)
Comment (geigerj) [inline generated/python/gapic-google-cloud-speech-v1beta1/setup.py]: Done

**Comment by geigerj** [comment]
`*.pyc` files removed and pattern added to .gitignore

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,29 @@
...
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+
+# Python build
+*.pyc
+*.pyo
```
Comment (bjwatson) [inline .gitignore]: There's a `generated/python/.gitignore`. Let's put Python-specific stuff there.

File: generated/python/grpc-google-cloud-speech-v1beta1/setup.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+import setuptools
+
+from setuptools import setup, find_packages
+
+install_requires = [
+  'oauth2client>=1.4.11
```
Comment (bjwatson) [inline generated/python/grpc-google-cloud-speech-v1beta1/setup.py]: Missing closing quote and comma. Anyway, can you change this to `>=3.0.0, <4.0.0dev`?

File: generated/python/grpc-google-cloud-speech-v1beta1/setup.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  'grpcio>=1.0.0, <2.0.0',
+  'googleapis-common-protos[grpc]>=1.3.4, <2.0.0'
```
Comment (bjwatson) [inline generated/python/grpc-google-cloud-speech-v1beta1/setup.py]: Add `dev` to the end of the upper bounds. Protects us against pre-releases.

File: generated/python/gapic-google-cloud-speech-v1beta1/setup.py [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+"""
+
+from setuptools import setup, find_packages
+import sys
+
+install_requires = [
+    'googleapis-common-protos>=1.3.4, <2.0.0',
+    'google-gax>=0.14.1, <0.15.0',
+    'grpc-google-cloud-speech-v1beta1>=0.10.0, <0.11.0',
+    'oauth2client>=1.4.11,
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-speech-v1beta1/setup.py]: Missing closing quote. Anyway, can you change this to `>=3.0.0, <4.0.0dev`?

File: generated/python/gapic-google-cloud-speech-v1beta1/setup.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+https://github.com/pypa/sampleproject
+"""
+
+from setuptools import setup, find_packages
+import sys
+
+install_requires = [
+    'googleapis-common-protos>=1.3.4, <2.0.0',
+    'google-gax>=0.14.1, <0.15.0',
+    'grpc-google-cloud-speech-v1beta1>=0.10.0, <0.11.0',
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-speech-v1beta1/setup.py]: Add `dev` to the end of the upper bounds. Protects us against pre-releases.

File: generated/python/gapic-google-cloud-speech-v1beta1/requirements.txt [hunk 9]
```diff
@@ -0,0 +1,4 @@
+googleapis-common-protos>=1.3.4, <2.0.0
+google-gax>=0.14.1, <0.15.0
+grpc-google-cloud-speech-v1beta1>=0.10.0, <0.11.0
+oauth2client>=1.4.11
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-speech-v1beta1/requirements.txt]: Update the same as `setup.py`

**Comment by bjwatson** [comment]
Superseded by https://github.com/googleapis/api-client-staging/pull/50

**Pull request closed without merging**